os/android/hwcservice.cpp

```diff
@@ -193,21 +209,26 @@ status_t HwcService::Controls::DisplayEnableBlank(uint32_t display,
 
 status_t HwcService::Controls::DisplayRestoreDefaultColorParam(
     uint32_t display, EHwcsColorControl color) {
-  // TO DO
+  hwcomposer::NativeDisplay *phyDisplay = mHwc.GetPrimaryDisplay();
```

Comment:
We don't do anything for extended displays ?

Reply:
do we need to support extended displays? I found other api implementations in this file also support physical display only.